source/glpiai/consumption.rst

```diff
@@ -0,0 +1,15 @@
+Consumption
+-----------
+
+It is possible to limit the use of the API by specifying a limit that must not be exceeded
+
+.. figure:: images/glpiai-8.png
+   :alt: see the usage
+   :scale: 100 %
+
+1.  In usage limit, indicate a value not to be exceeded. If this limit is accrossed, the plugin will no longer work until it is increased.
+2.  Indicate the real value of your API Key (https://platform.openai.com/usage). This value must be updated when the plugin usage data is not coordinated with the real API value.

```

Comment:
> This doesn't sound right, but I don't know what it is referring to in order to make a suggestion.

Hello Curtis, thanks for your feedback. I have tried to write this article without translation, not a good idea 😬. In the plugin, if you don't want to exceed a limit, you can setup one. there's a text already proposed but I didn't want to copy and paste it
![image](https://github.com/pluginsGLPI/doc/assets/153809566/fe65031d-0fff-4c41-8312-964cdafa7cda)

Reply:
If message is relevant, why not copy paste it?

Reply:
The 1st is not present in GLP, I wrote it, the 2nd is present but there's just one word that's been changed in fact

Reply:
I've just changed the text, does that make it clearer?

Reply:
No, and after looking at the plugin code I am still confused by the purpose of the second usage field. I can see that the custom usage is added when determining the total usage, but I don't know why it would be needed. Is it in case the API key is used outside of the AI plugin?

Reply:
Yes, I think that's right. This value has to be updated manually if it doesn't correspond to the actual usage value (in the case where, as you say, the API is used elsewhere). If I add a value to this field, the usage rate increases proportionally. 

![image](https://github.com/pluginsGLPI/doc/assets/153809566/a5906f88-e632-41f5-9542-63d5a72bb353)

Reply:
After more research, it looks like OpenAI doesn't have any way to check the tokens used per-API key. So this plugin keeps track of the tokens that the API responses say were used. I think this needs clarified better here. On the plugin side the tooltip for the usage "progress" says "Usage is linked only to this instance of the plugin. If plugin is uninstalled, usage data will be lost." which is OK.